packages/agoric-cli/test/agops-governance-smoketest.sh

```diff
@@ -10,7 +10,7 @@ if [ -z "$AGORIC_NET" ]; then
 yarn install && yarn build
 
 # local chain running with wallet provisioned
-packages/inter-protocol/scripts/start-local-chain.sh
+packages/proposals/scripts/inter-protocol/start-local-chain.sh
```

Comment:
This looks like an old search-and-replace that wasn't updated for `@agoric/builders`.  There are a few others in this package.

```suggestion
packages/builders/scripts/inter-protocol/start-local-chain.sh
```

Reply:
thx, fixed